config/helm/kustomization.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: '{{ .Release.Namespace }}'
+
+resources:
+  - input/deployment-rbac.yaml
+  - input/mic-exception.yaml
```

Comment:
Did we apply any modification to the files on the `input` folder? Could we use a link for those files in a release upstream?

Reply:
Good shout. No mods were made. All things considered, i'll incur the technical debt and sort this out in a smaller PR. Sound fair?

Reply:
Perfect!